feat: add angularjs framework (#1)
herodevs/endoflife.date: products/angular-js.md

    latestReleaseDate: 2016-10-11
    lts: false
    discontinued: true
    link: https://github.com/angular/angular.js/releases/tag/v1.2.32

-   releaseCycle: "1.1"
    releaseDate: 2012-08-13
    support: 2021-12-31
    eol: 2021-12-31
    extendedSupport: false
    latest: "1.1.5"
    latestReleaseDate: 2013-05-22
    lts: false
    discontinued: true
    link: https://github.com/angular/angular.js/releases/tag/v1.1.5

-   releaseCycle: "1.0"
    releaseDate: 2010-10-20
    support: 2021-12-31
    eol: 2021-12-31
    extendedSupport: false
    latest: "1.0.8"
    latestReleaseDate: 2013-08-22
    lts: false
    discontinued: true
    link: https://github.com/angular/angular.js/releases/tag/v1.0.8

---

> AngularJS is a free and open-source JavaScript-based web framework for developing single-page applications. It aims to simplify both the development and the testing of such applications by providing a framework for client-side model–view–controller (MVC) and model–view–viewmodel (MVVM) architectures, along with components commonly used in web applications and progressive web applications. AngularJS, referred to as Angular 1, was built and maintained by Google.

> AngularJS was [deprecated](https://docs.angularjs.org/misc/version-support-status) on December 31, 2021 after an [extended LTS period](https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c) on the final version 1.8 which was released June 4, 2020.

Reviewer: A couple of revisions here: 

1) Replace "an extended LTS period" with just "a LTS Period"
2) "final version 1.8 which was released June 4, 2020" change to "final version 1.8.3 which was released April 7, 2022"

Author: 1. Replace "an extended LTS period" with just "a LTS Period"
```suggestion
AngularJS was [deprecated](https://docs.angularjs.org/misc/version-support-status) on December 31, 2021 after a [LTS period](https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c) on the final version 1.8.3 which was released April 7, 2022.
```

## Extended Long Term Support

> The independent team at XLTS.dev offers [extended long term support for AngularJS](https://xlts.dev/angularjs?utm_source=endoflife.date&utm_medium=open-source-pr&utm_campaign=xlts-angularjs-blog&utm_content=pr-link). On January 1st 2022, the XLTS.dev team in [partnership with the AngularJS core team](https://docs.angularjs.org/misc/version-support-status) began maintaining a fork of the latest version of AngularJS and provides software updates that fit any of the following criteria:

Reviewer: Changes: 
Replace "the XLTS.dev team in [partnership with the AngularJS core team](https://docs.angularjs.org/misc/version-support-status) began maintaining a fork" - with -> "the XLTS.dev team in began maintaining, supporting and releasing a fork"

Author: ```suggestion
The independent team at XLTS.dev offers [extended long term support for AngularJS](https://xlts.dev/angularjs?utm_source=endoflife.date&utm_medium=open-source-pr&utm_campaign=xlts-angularjs-blog&utm_content=pr-link). On January 1st 2022, the XLTS.dev team began maintaining, supporting and realsing a fork of the latest version of AngularJS and provides software updates that fit any of the following criteria:
```
